records/2022-11-04.reliability.sms-limits.md

```diff
@@ -0,0 +1,58 @@
+# SMS Limits
+
+Date: 2022-11-04
+
+## Status
+
+**DRAFT**
+
+## Context
+
+Originally Notify only had a "combined" email and SMS limit. 
```

Comment:
```suggestion
Prior to changes brought in at the time of writing this ADR, Notify only had a "combined" email and SMS limit. As we need better individual control of each notification types, we implemented a limit to control the SMS limits associated with each service.

Note that the original limit still act as a global one for emails and SMS, i.e. a global limit of 10,000 and SMS limit of 1,000 on SMS for a service would allow the service to both send during a day 10,000 emails but 0 SMS or 9,000 emails and 1,000 SMS. We want to refine the global limit in the future to be exclusive to emails.
```